test/Test/Pos/Util/Limits.hs

```diff
@@ -0,0 +1,30 @@
+{-# LANGUAGE ScopedTypeVariables #-}
+
+-- | Test on functions trimming data to given limit.
+
+module Test.Pos.Util.Limits where
```

Comment:
Oh, nice catch. It's actually not a convention, but a restriction of hspec. I guess it skips those tests. I only ran them in ghci, so didn't notice that.